pkg/node/devnode.go

```diff
@@ -72,6 +72,7 @@ type DevBee struct {
 	errorLogWriter   io.Writer
 	apiServer        *http.Server
 	debugAPIServer   *http.Server
+	quitChunk        chan bool
```

Comment:
This should be called `quitErrPusher` or something. As you are replicating a pusher which returns error.